aether-common-module/aether/common/conf/urls.py

```diff
@@ -112,9 +124,12 @@ def generate_urlpatterns(token=False, kernel=False):  # pragma: no cover
         # assertions, a deployment configuration missing e.g. `AETHER_KERNEL_URL`
         # will seem to be in order until `get_kernel_server_url()` is called.
         if settings.TESTING:
-            msg = 'Environment variable "AETHER_KERNEL_URL_TEST" is not set'
+            msg = _('Environment variable "AETHER_KERNEL_URL_TEST" is not set')
         else:
-            msg = 'Environment variable "AETHER_KERNEL_URL" is not set'
+            msg = _('Environment variable "AETHER_KERNEL_URL" is not set')
         assert get_kernel_server_url(), msg
 
+        msg_token = _('Environment variable "AETHER_KERNEL_TOKEN" is not set')
+        assert get_kernel_server_token(), msg_token
```

Comment:
But isn't this an odd place to put these asserts? I wouldn't think to look for them in `generate_urlpatterns`

Reply:
But it's the only place where you can indicate that you need access to kernel. I agree that's a weird place and I want to refactor this and move it to another place but... no time so far. :(